File: src/main/java/com/starshootercity/abilities/FlameParticles.java
package com.starshootercity.abilities;

import net.kyori.adventure.key.Key;
import org.bukkit.Particle;
import org.jetbrains.annotations.NotNull;

public class FlameParticles implements ParticleAbility {
    @Override
    public @NotNull Key getKey() {
        return Key.key("origins:flame_particles");
    }

    @Override
    public Particle getParticle() {
        return Particle.FLAME;
    }

    @Override
    public int getFrequency() {
        return 4;
    }
}

File: core/src/main/java/com/starshootercity/abilities/FlameParticles.kt
package com.starshootercity.abilities

import net.kyori.adventure.key.Key
import org.bukkit.Particle

class FlameParticles : ParticleAbility {
    override fun getKey(): Key {
        return Key.key("origins:flame_particles")
    }

    override fun getParticle(): Particle {
        return Particle.FLAME
    }

    override fun getFrequency(): Int {
        return 4
    }
}
